portfolio/src/main/java/com/google/sps/servlets/ImageServlet.java

```diff
@@ -14,13 +14,13 @@
 
 package com.google.sps.servlets;
 
-import com.google.appengine.api.blobstore.BlobstoreService;
-import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
```

Comment:
Why change the order of the import here?

Reply:
I figured out how to have VSCode autoformat to the exact Google Java style guide, so it reshuffled the imports to comply with them.